lib/templates.ts

```diff
@@ -1562,3 +1563,86 @@ export const checkIfClosed = async (formId: string) => {
     return null;
   }
 };
+
+export const getFormJSONConfig = async (formId: string) => {
+  await authorization.canEditForm(formId).catch((e) => {
+    logEvent(
+      e.user.id,
+      { type: "Form", id: formId },
+      "AccessDenied",
+      "Attempted to update security attribute"
+    );
+    throw e;
+  });
+
+  const result = await prisma.template
+    .findUnique({
+      where: { id: formId },
+      select: { jsonConfig: true },
+    })
+    .catch((e) => prismaErrors(e, null));
+
+  if (!result) {
+    throw new Error("Template not found");
+  }
+
+  let jsonConfig: FormProperties;
+  const raw = result.jsonConfig;
+
+  if (typeof raw === "string") {
+    // Only parse if (unexpectedly) stored as a string
+    jsonConfig = JSON.parse(raw) as FormProperties;
+  } else {
+    jsonConfig = raw as FormProperties;
+  }
+
+  return jsonConfig;
+};
+
+export const updateFormJSONConfig = async (formID: string, jsonConfig: FormProperties) => {
+  const { user } = await authorization.canEditForm(formID).catch((e) => {
+    logEvent(
+      e.user.id,
+      { type: "Form", id: formID },
+      "AccessDenied",
+      "Attempted to update security attribute"
+    );
+    throw e;
+  });
+
+  // Ensure it is a pure JSON value
+  const sanitized = JSON.parse(JSON.stringify(jsonConfig)) as Prisma.JsonObject;
```

Comment:
As mentioned in Slack we should probably validate the JSON object with the template schema.

Here is an existing function that does it https://github.com/cds-snc/platform-forms-client/blob/a9de7c5e68a51d826d6d36c66499ef2d223c3cdb/lib/templates.ts#L503